tests/migrations-rollback

```diff
@@ -7,8 +7,8 @@ MIGRATIONS_ORDER=`mktemp /tmp/migrations.order.XXXXXX`
 ./manage.py migrations_order > $MIGRATIONS_ORDER
 cat $MIGRATIONS_ORDER
 
-echo "..... Apply all migrations"
-./manage.py migrate
+echo "..... Perform Kiwi TCMS upgrade"
```

Comment:
Revert back the info text. We're not performing upgrade here, just using the `upgrade` command in a slightly illegal way.